src/main/scala/de/upb/cs/swt/delphi/instanceregistry/daos/DatabaseAuthDAO.scala

```diff
@@ -122,7 +167,36 @@ class DatabaseAuthDAO (configuration : Configuration) extends AuthDAO with AppLo
     }
   }
 
+  private def removeAllUsers(): Unit = {
```

Comment:
My local IDE says this private method is unused. Please remove it if it is not necessary for some future use-case.